specs/core/1_beacon-chain-misc.md

```diff
@@ -0,0 +1,238 @@
+# Phase 1 miscellaneous beacon chain changes
+
+## Table of contents
+
+<!-- TOC -->
+
+- [Phase 1 miscellaneous beacon chain changes](#phase-1-miscellaneous-beacon-chain-changes)
+    - [Table of contents](#table-of-contents)
+    - [Configuration](#configuration)
+    - [Containers](#containers)
+            - [`CompactCommittee`](#compactcommittee)
+            - [`ShardReceiptProof`](#shardreceiptproof)
+    - [Helper functions](#helper-functions)
+            - [`pack_compact_validator`](#pack_compact_validator)
+            - [`unpack_compact_validator`](#unpack_compact_validator)
+            - [`committee_to_compact_committee`](#committee_to_compact_committee)
+            - [`get_previous_power_of_2`](#get_previous_power_of_2)
+            - [`verify_merkle_proof`](#verify_merkle_proof)
+            - [`compute_historical_state_generalized_index`](#compute_historical_state_generalized_index)
+            - [`get_generalized_index_of_crosslink_header`](#get_generalized_index_of_crosslink_header)
+            - [`process_shard_receipt_proof`](#process_shard_receipt_proof)
+    - [Changes](#changes)
+        - [Phase 0 container updates](#phase-0-container-updates)
+            - [`BeaconState`](#beaconstate)
+            - [`BeaconBlockBody`](#beaconblockbody)
+        - [Persistent committees](#persistent-committees)
+        - [Shard receipt processing](#shard-receipt-processing)
+
+<!-- /TOC -->
+
+## Configuration
+
+| Name | Value | Unit | Duration
+| - | - | - | - |
+| `MAX_SHARD_RECEIPT_PROOFS` | `2**0` (= 1) | - | - |
+| `PERSISTENT_COMMITTEE_ROOT_LENGTH` | `2**8` (= 256) | periods | ~9 months |
+| `MICRO_REWARD` | `Gwei(2**0)` (=1) | Gwei | - |
```

Comment:
I added a stub number here. @vbuterin could you give a more precise number? Or we should use `PLACEHOLDER` here?

Reply:
See suggestions; I basically set it to be the same as the reward for including chunk challenge responses.